src/api/v1/dataset_rows.py

```diff
@@ -22,13 +28,33 @@
 )
 async def create_dataset_rows(
     body: List[DatasetRowCreate],
+    background_tasks: BackgroundTasks,
     db: Session = Depends(get_db),
     authenticated_user: User = Depends(authenticate_user),
 ) -> DatasetRow:
 
     model = crud.models.get(db=db, _id=dict(body[0])["model_id"])
     if model:
         new_dataset_rows = crud.dataset_rows.create_many(db=db, obj_list=body)
+        # TODO Use returned dataset_rows and not body
+        inserted_dataset_rows = jsonable_encoder(body)
+        processed_dataset_rows = [x["processed"] for x in inserted_dataset_rows]
+        processed_dataset_rows_pd = pd.DataFrame(processed_dataset_rows)
```

Comment:
@momegas For some reason the crud action returns an array with empty objects, although later at line 58 the object is return correctly by the endpoint.

Reply:
Just to for keeping context, the `create_all` function needed to refresh the inserted object